play/plugin/build.gradle.kts

```diff
@@ -59,23 +64,16 @@ tasks.withType<Test> {
 }
 
 gradlePlugin {
+    website.set("https://github.com/Triple-T/gradle-play-publisher")
+    vcsUrl.set("https://github.com/Triple-T/gradle-play-publisher")
+
     plugins.create("play") {
         id = "com.github.triplet.play"
         displayName = "Gradle Play Publisher"
         description = "Gradle Play Publisher allows you to upload your App Bundle or APK " +
                 "and other app details to the Google Play Store."
         implementationClass = "com.github.triplet.gradle.play.PlayPublisherPlugin"
-    }
-}
-
-pluginBundle {
-    website = "https://github.com/Triple-T/gradle-play-publisher"
-    vcsUrl = "https://github.com/Triple-T/gradle-play-publisher"
-    tags = listOf("android", "google-play", "publishing", "deployment", "apps", "mobile")
-
-    mavenCoordinates {
```

Comment:
I been forced to bump `gradlePublish` version to `1.1.0` to be compatible with Gradle 8 to address [this](https://github.com/Triple-T/gradle-play-publisher/actions/runs/4864010574/jobs/8672470785#step:7:174) and as far I know, the `mavenCoordinates` closure is not available anymore

Reply:
`website`, `vcsUrl` and `tags` are now part of the  `gradlePlugin` closure

Reply:
What happened to `groupId` and `artifactId`? I feel like publishing will probably fail without them.

Reply:
From the [docs](https://plugins.gradle.org/docs/publish-plugin#breaking-changes):
> #### Breaking changes:
> The mavenCoordinates block is no longer defined and can’t be used to override publication GAV coordinates. The coordinates are always based on the Maven publication’s GAV (which defaults to the Project’s group, name & version).

Reply:
I also noted that `publishToMavenLocal` is broken and generates a POM with an invalid `plugin` artifactId in its dependency. I can make it work by removing any explicit `artifactId` set in the POM configuration and replace the `plugin` project's name with `play-publisher` in the settings